### PR TITLE
CVM: Support for CMK through DES

### PR DESCRIPTION
**Pull request by kamalca**

### CODE REVIEW

File: lisa/features/security_profile.py [hunk 1]
```diff
@@ -59,6 +59,12 @@ class SecurityProfileSettings(schema.FeatureSettings):
         ),
     )
     encrypt_disk: bool = field(default=False)
+    disk_encryption_set_id: str = field(
```
Comment (squirrelsc) [inline lisa/features/security_profile.py]: If it's an Azure specified setting, it should be defined in Azure's feature settings.

Comment (kamalca) [reply lisa/features/security_profile.py]: I'm having trouble moving this field. _call_requirement_method() needs access to the field but moving the method to Azure also doesn't seem to work. Please advise.

Comment (squirrelsc) [reply lisa/features/security_profile.py]: Azure should inherit this method, and do extra logic after called the super one.

Comment (kamalca) [reply lisa/features/security_profile.py]: Fixed

**Review by squirrelsc — APPROVED** [review]

**Pull request merged**